### PR TITLE
Extra information in DataAnnotations localization

### PR DESCRIPTION
**Pull request by sintetico82**
I try to esxplain the solution for use SharedResources class in Data Annotation, thanks to the comment of this issue https://github.com/aspnet/Localization/issues/286

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@sintetico82  I edited your contribution. Can you review my changes?

**Comment by Rick-Anderson** [comment]
@audacity76  @hishamco @simon25608 please review.

File: aspnetcore/fundamentals/localization.md [hunk 1]
```diff
@@ -98,6 +98,24 @@ DataAnnotations error messages are localized with `IStringLocalizer<T>`. Using t
...
+### Using one resource string for multiple classes
+
+The following code shows how to use one resource string for validation attributes with multiple classes:
+
+```
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddMvc()
+        .AddDataAnnotationsLocalization(options => {
+            options.DataAnnotationLocalizerProvider = (type, factory) =>
```
Comment (hishamco) [inline aspnetcore/fundamentals/localization.md]: It would be nice if you highlight this line

Comment (Rick-Anderson) [inline aspnetcore/fundamentals/localization.md]: @hishamco  You can only highlight when you import a snippet. Inline code can't be highlighted.

Comment (hishamco) [inline aspnetcore/fundamentals/localization.md]: 👍

**Comment by m4ss1m0g** [comment]
With this solution the application will use single resx. Why do not you make _"NuGet-table"_ the `DataAnnotation `messages resx and therefore customizable within the application ?

**Comment by apis3445** [comment]
Do you have a github sample for how I can localize the data annotations messages with one file?

**Comment by hishamco** [comment]
@apis3445 you can check out the [Localization.Starter.Web](https://github.com/aspnet/Entropy/tree/dev/samples/Localization.StarterWeb)

**Pull request merged**